crdppf/lib/content.py

```diff
@@ -7,7 +7,7 @@
 from crdppf.lib.wmts_parsing import wmts_layer
 from crdppf.lib.geometry_functions import get_feature_bbox, get_print_format, get_feature_center
 
-from crdppf.views.get_features import get_features_function
+from crdppf.util.get_feature_functions import get_features_function

```

Comment:
It a bit weird to have <pre>get_feature_function<b>s</b></pre> as file name and <pre>get_feature<b>s</b>_function</pre> as function name. Isn't it ?

Reply:
That's historical... I'll might change it if we keep v1 longer